private_sharing/utilities.py

```diff
@@ -50,3 +53,22 @@ def source_to_url_slug(source):
         if match:
             project = DataRequestProject.objects.get(id=int(match.group('id')))
             return project.slug
+
+
+def send_withdrawal_email(project, slug):
+    """
+    Email a project to notify them that a member has withdrawn.
+    """
+
+    params = {
+        "withdrawn_url": full_url("/erase-member-data/{0}".format(project.slug)),
```

Comment:
Ah, the URL as constructed here isn't working. Maybe passing the project_member to this template would make more sense..

And maybe you could use something like `reverse_lazy('direct-sharing:erase-member-data', kwargs={'slug': project_member.project.slug})`?

Reply:
huh, it seems to work for me?  I click on it, and it comes up.